src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -83,6 +83,8 @@
         .packageVersionBound(packageConfig.generatedPackageVersionBound(language))
         .protoPath(packageConfig.protoPath())
         .shortName(packageConfig.shortName())
+        .gapicConfigName(packageConfig.shortName() + "_gapic.yaml")
```

Comment:
ISTM that since `artman` know the "real" value of the GAPIC config name, it should pass that value to toolkit, rather than toolkit guessing the name based on a heuristic. Then again, I don't know what this value actually does Java codegen...